[2024-03-09T17:58:05.233+0530] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: python_operator_dag.taskA scheduled__2024-03-08T00:00:00+00:00 [queued]>
[2024-03-09T17:58:05.238+0530] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: python_operator_dag.taskA scheduled__2024-03-08T00:00:00+00:00 [queued]>
[2024-03-09T17:58:05.239+0530] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-09T17:58:05.265+0530] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): taskA> on 2024-03-08 00:00:00+00:00
[2024-03-09T17:58:05.267+0530] {standard_task_runner.py:60} INFO - Started process 6735 to run task
[2024-03-09T17:58:05.272+0530] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'python_operator_dag', 'taskA', 'scheduled__2024-03-08T00:00:00+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/execute_python_operators.py', '--cfg-path', '/tmp/tmped_vtbdd']
[2024-03-09T17:58:05.273+0530] {standard_task_runner.py:88} INFO - Job 63: Subtask taskA
[2024-03-09T17:58:05.341+0530] {task_command.py:423} INFO - Running <TaskInstance: python_operator_dag.taskA scheduled__2024-03-08T00:00:00+00:00 [running]> on host BreakThrough.
[2024-03-09T17:58:05.440+0530] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kai' AIRFLOW_CTX_DAG_ID='python_operator_dag' AIRFLOW_CTX_TASK_ID='taskA' AIRFLOW_CTX_EXECUTION_DATE='2024-03-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-08T00:00:00+00:00'
[2024-03-09T17:58:05.442+0530] {logging_mixin.py:188} INFO - Hello World
[2024-03-09T17:58:05.442+0530] {python.py:202} INFO - Done. Returned value was: None
[2024-03-09T17:58:05.446+0530] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=python_operator_dag, task_id=taskA, execution_date=20240308T000000, start_date=20240309T122805, end_date=20240309T122805
[2024-03-09T17:58:05.486+0530] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-09T17:58:05.498+0530] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
